pkg/log/log.go

```diff
@@ -112,6 +119,10 @@ func (l *OsqueryLogAdapter) logInfoAboutUnrecognizedProcessLockingPidfile(p []by
 		return
 	}
 
+	l.runAndLogPs(pid)
+	l.runAndLogLsofByPID(pid)
+	l.runAndLogLsofOnPidfile()
+
 	unknownProcess, err := process.NewProcess(int32(pid))
```

Comment:
Thank you for adding the timeouts. It occurs to me -- this is in the logging path, and we're blocking logs on execs. Thoughts about a goroutine for something in here? (Maybe just the call to `logInfoAboutUnrecognizedProcessLockingPidfile`)

Reply:
I was wondering about that -- yeah, that makes sense to me! I'll update and test.